Adding initial event info for 2020-09-29 (#320)
RSE-leaders/SORSE20: _events/talks/event-ID_UNKNOWN.md

---

> title: "Digital Humanities RSE: King’s Digital Lab as experiment and lifecycle"

Reviewer: ```suggestion
title: "Digital Humanities RSE: King's Digital Lab as experiment and lifecycle"
```

authors:
    - &speaker
      name:  James Smithies
      is_speaker: true
      affiliation: 1
    - name:  Arianna Ciula
      is_speaker: true
      affiliation: 1
affiliations:

>     - name: King’s Digital Laboratory, King's College London, London, UK

Reviewer: ```suggestion
    - name: King's Digital Laboratory, King's College London, London, UK
```

>     - name: King's Digital Laboratory, King's College London, London, UK

Reviewer 2: ```suggestion
    - name: King's Digital Laboratory, King's College London, UK
```

      index: 1
time:
    - start: 2020-09-29T15:00:00Z
      end: 2020-09-29T16:30:00Z
author: *speaker
category: talks
language: English
date: 2020-09-08
last_modified_at: 2020-09-08

> registration_url: https://indico.scc.kit.edu/event/ID_UNKNOWN/

Reviewer 2: ```suggestion
registration_url: https://indico.scc.kit.edu/event/923/
```

---

> This SORSE event describes King’s Digital Lab (KDL), a Research Software Engineering lab operating within the Faculty of Arts and Humanities at King’s College London (UK). The KDL team of 18 project managers, analysts, designers, engineers, and systems managers specialise in arts & humanities, cultural heritage, and creative industries research and development. The talk will provide a current state overview of the lab, and describe our RSE HR roles (see [https://zenodo.org/record/2564790](https://zenodo.org/record/2564790)) and a relatively recent trial initiative that defines the different ways the team can contribute to research. A more technically-oriented overview of our ongoing work in refining the fundamental components of  KDL Software Development Lifecycle (see https://github.com/kingsdigitallab/sdlc-for-rse/) will also be provided, including descriptions of our modelling workflows, processes, and efforts to increase sustainability, reproducibility, and research data management best practice ([https://zenodo.org/record/3361580#.X09x6mdKhhE](https://zenodo.org/record/3361580#.X09x6mdKhhE)).

Reviewer: ```suggestion
This SORSE event describes King's Digital Lab (KDL), a Research Software Engineering lab operating within the Faculty of Arts and Humanities at King's College London (UK). The KDL team of 18 project managers, analysts, designers, engineers, and systems managers specialise in arts & humanities, cultural heritage, and creative industries research and development. The talk will provide a current state overview of the lab, and describe our RSE HR roles (see [https://zenodo.org/record/2564790](https://zenodo.org/record/2564790)) and a relatively recent trial initiative that defines the different ways the team can contribute to research. A more technically-oriented overview of our ongoing work in refining the fundamental components of  KDL Software Development Lifecycle (see https://github.com/kingsdigitallab/sdlc-for-rse/) will also be provided, including descriptions of our modelling workflows, processes, and efforts to increase sustainability, reproducibility, and research data management best practice ([https://zenodo.org/record/3361580#.X09x6mdKhhE](https://zenodo.org/record/3361580#.X09x6mdKhhE)).
```

### About the speakers

> **James Smithies** is Director of King’s Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.

Reviewer: ```suggestion
**James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```

> **James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King’s College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.

Reviewer: ```suggestion
**James Smithies** is Director of King's Digital Lab & Reader and Deputy Director of eResearch at King's College London. He was previously Senior Lecturer in Digital Humanities and Associate Director of the UC CEISMIC Digital Archive at the University of Canterbury, New Zealand. He has worked in the government and commercial IT sectors as a technical writer and editor, business analyst, and project manager. In 2017 he published a monograph for Palgrave Macmillan titled [Digital Humanities and the Digital Modern](https://www.palgrave.com/gp/book/9781137499431). James is interested in applied and theoretical digital humanities, including data analysis, minimal computing, digital archives, modelling and virtual worlds. He has subsidiary interests in critical and media theory, and postphenomenology. His historical work focuses on the history of ideas, technology (including but not limited to computing), and literature. See his [personal website](https://jamessmithies.org/) for more information about his work.
```

> **Arianna Ciula** is Deputy Director & Senior Research Software Analyst at King’s Digital Lab. She has broad experience in digital humanities research and teaching, research management, and digital research infrastructures. She holds a PhD in Manuscript and Book Studies (digital palaeography, University of Siena), an MA in Applied Computing in the Humanities (King’s College London) and a BA Hons in Communication sciences (computational linguistics, University of Siena). She worked at King’s in the past as Research Associate (Centre for Computing in the Humanities, 2003-2009). From 2009 to 2012, she worked as Science officer at the European Science Foundation (Humanities) and from 2013 she worked as Research Facilitator at the University of Roehampton for three years..

Reviewer: ```suggestion
**Arianna Ciula** is Deputy Director & Senior Research Software Analyst at King's Digital Lab. She has broad experience in digital humanities research and teaching, research management, and digital research infrastructures. She holds a PhD in Manuscript and Book Studies (digital palaeography, University of Siena), an MA in Applied Computing in the Humanities (King's College London) and a BA Hons in Communication sciences (computational linguistics, University of Siena). She worked at King's in the past as Research Associate (Centre for Computing in the Humanities, 2003-2009). From 2009 to 2012, she worked as Science officer at the European Science Foundation (Humanities) and from 2013 she worked as Research Facilitator at the University of Roehampton for three years..
```

> Her personal research interests focus on the modelling of scholarly digital resources related to primary sources. She lectured and published on humanities computing, in particular on digital manuscript studies and editing; she has organised conferences and workshops in digital humanities, and is an active member of its international community. She was co-PI in the project ‘[Modelling between Digital and Humanities: Thinking in Practice](http://modellingdh.eu/)’ funded by the Volkswagen Foundation (2016-18). See list of [publications](https://kclpure.kcl.ac.uk/portal/en/persons/arianna-ciula(8b156ef5-c75d-46a7-9be9-2eb2a4666f31)/publications.html).

Reviewer: ```suggestion
Her personal research interests focus on the modelling of scholarly digital resources related to primary sources. She lectured and published on humanities computing, in particular on digital manuscript studies and editing; she has organised conferences and workshops in digital humanities, and is an active member of its international community. She was co-PI in the project "[Modelling between Digital and Humanities: Thinking in Practice](http://modellingdh.eu/)" funded by the Volkswagen Foundation (2016-18). See list of [publications](https://kclpure.kcl.ac.uk/portal/en/persons/arianna-ciula(8b156ef5-c75d-46a7-9be9-2eb2a4666f31)/publications.html).
```
